gallery/src/components/NxVulnerabilityIndicator/NxVulnerabilityIndicatorPage.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxVulnerabilityIndicatorExample from './NxVulnerabilityIndicatorExample';
+import NxVulnerabilityIndicatorColumnExample from './NxVulnerabilityIndicatorColumnExample';
+import NxVulnerabilityIndicatorGridExample from './NxVulnerabilityIndicatorGridExample';
+
+const nxVulnerabilityIndicatorCode = require('./NxVulnerabilityIndicatorExample?raw'),
+    nxVulnerabilityIndicatorColumnCode = require('./NxVulnerabilityIndicatorColumnExample?raw'),
+    nxVulnerabilityIndicatorGridCode = require('./NxVulnerabilityIndicatorGridExample?raw');
+
+const NxVulnerabilityIndicatorPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        Basic style for small indicator which displays a number representing a count of violations, and a short pre-set
+        text string which displays the vulnerability severity.
+      </p>
+      <p className="nx-p">
+        Three basic layouts have been demonstrated below. It is expected that one of these should satisfy any
+        requirment.
+      </p>
+      <table className="nx-table nx-table--gallery-props">
+        <thead>
+          <tr className="nx-table-row">
+            <th className="nx-cell nx-cell--header">Class</th>
+            <th className="nx-cell nx-cell--header">Location</th>
+            <th className="nx-cell nx-cell--header">Details</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-vulnerability-<br />indicator-container</code></td>
+            <td className="nx-cell">Container</td>
+            <td className="nx-cell">Basic container class. It's always used with one of the modifiers below.</td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell">
+              <code className="nx-code">.nx-vulnerability-<br />indicator-container--row</code>
+            </td>
+            <td className="nx-cell">
+              Modifier of <code className="nx-code">.nx-vulnerability-<br />indicator-container</code>
+            </td>
+            <td className="nx-cell">
+              Used when you want the indicators to appear in a row. This is the default. If no layout is specified
+              "row" will be used.
```

Comment:
What's that point of having a separate class for this if it is the default?

Reply:
> What's that point of having a separate class for this if it is the default?

There's so much to override between them I prefer a separate class.

Reply:
> > What's that point of having a separate class for this if it is the default?
> 
> There's so much to override between them I prefer a separate class.

Though now with the move to grid it makes more sense to remove the custom class: https://github.com/sonatype/sonatype-react-shared-components/pull/343/commits/d738dd31f8092cde9df5949377cbda7b6f5b1052